infra/aws/terraform/audit-account/securityhub.tf

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+resource "aws_securityhub_organization_admin_account" "default" {
+  admin_account_id = local.audit-account-id
+}
+
+resource "aws_securityhub_organization_configuration" "default" {
+  auto_enable = true
+}
+
+resource "aws_securityhub_account" "this" {
+  provider = aws.audit
+}
+
+# Enable Standard: AWS Foundational Security Best Practices
```

Comment:
Eventually, we'll also want to manage Security Hub in member accounts.

Optionally, take a list / set of controls to _disable_ as a variable to the appropriate module. For example, we might _not_ want&hellip;

- AWS Foundational Security Best Practices: S3 buckets should prohibit public read access

Reply:
Implementation will done in a separate PR.